# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../ippool/ippool-1.3-include-limits-h.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ippool-1.3/usl/usl_timer.c.orig	2011-08-02 18:51:42.166256696 +0200
+++ ippool-1.3/usl/usl_timer.c	2011-08-02 18:52:16.839242988 +0200
@@ -42,6 +42,7 @@
 #include <signal.h>
 #include <string.h>
 #include <sys/time.h>
+#include <limits.h>
 
 #include "usl.h"
 
